book/chapters/chapter2/data_and_basic_modeling.qmd

```diff
@@ -392,18 +392,18 @@ To access the hyperparameters in the decision tree, we use `r index("$param_set"
 lrn_rpart$param_set
 ```
 
-The output above is a `r ref("ParamSet", aside = TRUE)` object.
-These objects provide information on hyperparameters including their name (`id`), data type (`class`), acceptable ranges for hyperparameter values (`lower`, `upper`), the number of levels possible if the data type is categorical (`nlevels`), the default value from the underlying package (`default`), and finally the set value (`value`).
+The output above is a `r ref("ParamSet", aside = TRUE)` object, supplied by the `r ref_pkg("paradox")` package.
+These objects provide information on hyperparameters including their name (`id`), data type (`class`), technically valid ranges for hyperparameter values (`lower`, `upper`), the number of levels possible if the data type is categorical (`nlevels`), the default value from the underlying package (`default`), and finally the set value (`value`).
 The second column references classes defined in `r ref_pkg("paradox")` that determine the class of the parameter and the possible values it can take.
 @tbl-parameters-classes lists the possible hyperparameter types, all of which inherit from `r ref("Param")`.
 
 | Hyperparameter Class | Description                          |
 | -----------------  | ---------------------------------- |
-| `r ref("ParamDbl", index = TRUE)`  | Real-valued (Numeric) Parameters     |
-| `r ref("ParamInt", index = TRUE)`  | Integer Parameters                   |
-| `r ref("ParamFct", index = TRUE)`  | Categorical (Factor) Parameters      |
-| `r ref("ParamLgl", index = TRUE)`  | Logical / Boolean Parameters         |
-| `r ref("ParamUty", index = TRUE)`  | Untyped Parameters                   |
+| `r ref("ParamDbl", index = TRUE)`  | Real-valued (numeric) parameters     |
+| `r ref("ParamInt", index = TRUE)`  | Integer parameters                   |
+| `r ref("ParamFct", index = TRUE)`  | Categorical (factor) parameters      |
+| `r ref("ParamLgl", index = TRUE)`  | Logical / Boolean parameters         |
+| `r ref("ParamUty", index = TRUE)`  | Untyped parameters                   |
```

Comment:
Maybe change column 2 title to "Parameter Type" and then you don't need to write "parameters" each time in the columns?